Svg.Tests.Win/SvgClipPathTest.cs

```diff
@@ -68,4 +68,18 @@ public void WhenGroupWithClipAndChildWithMatrixRenderer_CanvasIsSavedCorrectly()
         c.AssertAreSimilar(97f, svgPath);
 
     }
+
+    [Test]
+    public void CreatingAClip_WithARectangle_GetClipBounds()
```

Comment:
please come up with a better name.
Tests should (please) be named like:
`<operation/action>_<precondition>_<expectedoutcome>`

So like
`WhenCalculatingBounds_AndRectangleIsClippedWhereClipIsSmallerThanTheRectangle_ReturnsTheClipBounds`